packages/terra-badge/tests/nightwatch/badge-spec.js

```diff
@@ -1,57 +0,0 @@
-/* eslint-disable no-unused-expressions */
```

Comment:
I agree, nothing needs tested in nightwatch now. With the removal of this spec, I think we make our first step towards our testing migration and move the nightwatch examples and routes into their own 'examples' directory. since both site, wdio tests and nightwatch tests will be sharing them. 

My reasoning why now is keeping them in nightwatch could be confusing later on since we are removing this file.

Reply:
👍

Reply:
Still think it would be good to make our first step towards our testing migration and move the nightwatch examples and routes into their own 'examples' directory. 

Other than that, this PR looks good to me!

Reply:
I'm not sure its worth the work... we have to move the files and update the terra-site doc. Being @bjankord is starting work to move components to storybook this is a short lived effort..

Reply:
sure